pom.xml

```diff
@@ -293,6 +294,67 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-enforcer-plugin</artifactId>
+                  <dependencies>
+                    <dependency>
+                      <groupId>de.skuzzle.enforcer</groupId>
+                      <artifactId>restrict-imports-enforcer-rule</artifactId>
+                      <version>${restrict-imports.enforcer.version}</version>
+                    </dependency>
+                  </dependencies>
```

Comment:
Nice! When i started looking at the changes, i immediately came here to see if we are going to restrict the import.

Reply:
Straight copy-paste from Phoenix :)